lib/src/controls/form/auto_suggest_box.dart

```diff
@@ -807,9 +814,10 @@ class _AutoSuggestBoxOverlayState extends State<_AutoSuggestBoxOverlay> {
                   itemBuilder: (context, index) {
                     final item = sortedItems[index];
                     return _AutoSuggestBoxOverlayTile(
-                      text: item.child ?? Text(item.value),
+                      text: item.child ?? Text(item.label),
                       selected: item._selected,
-                      onSelected: () => widget.onSelected(item),
+                      onSelected: () =>
+                          widget.onSelected(item as AutoSuggestBoxItem<T>),
```

Comment:
this casting is unecessary since sorter is now typed

Reply:
Fixed

Reply:
I've added some modifications because AFAIK the static sorter cannot be generic, but with my changes is now working